Changelog.md

```diff
@@ -3,6 +3,8 @@ __New Features__
 - Updates to OpenStudio 3.4.0/EnergyPlus 22.1.
 - Expanded capabilities for scheduling:
   - Allows modeling detailed HVAC setpoints via a schedule CSV file.
+  - Allows modeling detailed tank and heat pump water heater setpoints via a schedule CSV file.
```

Comment:
Doesn't apply for tankless water heaters.

Reply:
Are you confusing this with tank mode? Surely you can use detailed water heater setpoints for any type of water heater.

Reply:
No, I'm not. We're passing `@schedules-file` only into `apply_tank` and `apply_heat_pump`: https://github.com/NREL/OpenStudio-HPXML/blob/wh-geb/HPXMLtoOpenStudio/measure.rb#L1400-L1406.

I also confirmed this with @jmaguire1: "there is really no reason to allow [scheduled setpoints] for [tankless water heaters], you don't get any load shifting benefit from scheduling a tankless."

Reply:
I see. I understand the rationale, but suggest we allow detailed setpoints for any water heater type anyway. It's simpler, it's a small effort, and it causes no harm. You never know what someone might want to use it for. 

(If there _was_ a strong justification to not allow it, then at a minimum we have to document it better because there's nothing in the documentation right now saying that it only applies to certain water heater types.)

Reply:
For combi, too?

Reply:
Yes. Any water heating system that allows a [simple setpoint temperature](https://github.com/NREL/OpenStudio-HPXML/blob/f44aec2615d402c80afff028bdd0317c7362fb98/HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.xml#L1413) should be allowed a detailed setpoint schedule.